probes/pkg/ping/client/client.go

```diff
@@ -21,50 +21,91 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/perf-tests/probes/pkg/common"
+
 	"k8s.io/klog"
-	"k8s.io/perf-tests/probes/pkg/metrics"
 )
 
 var (
+	// inClusterNetworkLatency implements the In-Cluster Network Programming SLI, see
```

Comment:
can we move it to metrics.go in this directory?

Reply:
Sure, but why?

Reply:
We're generally trying to define metrics in metrics.go files in all kubernetes - it's just easier to find metrics then.

Reply:
sgtm

Is is documented/defined/written somewhere?

Reply:
not that I'm aware of